pkg/cmd/server/start/start_master.go

```diff
@@ -633,6 +633,27 @@ func (i genericInformers) ForResource(resource schema.GroupVersionResource) (kin
 	return nil, firstErr
 }
 
+func collectInitializers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) (map[string]origincontrollers.InitFunc, error) {
```

Comment:
yeah, I put it in as part of the stab at a test.  I'm certainly willing to pull all of that out.